event.html

```diff
@@ -59,34 +59,34 @@
     <div class="container">
         <div class="jumbotron event-jumbo mt-5 rounded">
 
-            <h2 class="p-2">
+            <h2 class="p-3">
                 SNBRN pool side at<br>phoenix hotel
             </h2>
             <img src="IMG/SNBRN.jpeg" class="h-50 mx-auto d-block">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
         </div>
 
         <div class="jumbotron event-jumbo mt-5">
-            <h2 class="p-2">
+            <h2 class="p-3">
                 wax motif poolside at<br>phoenix hotel
             </h2>
             <img src="IMG/Wax-motif.jpeg" class=" h-50 mx-auto d-block ">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end ">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
         </div>
 
         <div class="jumbotron event-jumbo mt-5">
-            <h2 class="p-2">
+            <h2 class="p-3">
                 chris lorenzo pool side at<br>phoenix hotel
             </h2>
             <img src="IMG/Chris-lorenzo.jpeg" class="h-50 mx-auto d-block">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end  ">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
```

Comment:
Since you're repeating this code in multiple places `btn btn-light col-2 etc` this is usually a good sign to abstract all of this into a reusable class. That way if you change the design later on you only change it one place instead of multiple places.